sdk/lib/crt/stdlib/qsort.c

```diff
@@ -91,7 +93,8 @@ __cdecl
 qsort(void *a, size_t n, size_t es, int (__cdecl *cmp)(const void*, const void*))
 {
 	char *pa, *pb, *pc, *pd, *pl, *pm, *pn;
-	int d, r, swaptype, swap_cnt;
+	int swaptype, swap_cnt;
+    size_t d, r;
```

Comment:
I think tabulation should be here instead of 4 spaces, according to the code style in this file.